scripts/bootstrap.py

```diff
@@ -113,6 +119,7 @@ def main():
         marathon_jenkins_port = os.environ['PORT1']
         mesos_master = os.environ['JENKINS_MESOS_MASTER']
         ssh_known_hosts = os.environ['SSH_KNOWN_HOSTS']
+        jenkins_strict_mode = os.environ['JENKINS_STRICT_MODE']
```

Comment:
Let's omit setting this variable and just set JENKINS_AGENT_USER iff user has specified strict mode in the package configuration. If JENKINS_AGENT_USER isn't set, let's leave it at the default (root).